### PR TITLE
[PUB-1195] Preventing Publish from crashing on failing updates

### PR DESCRIPTION
**Pull request by MayaUribe**
### Purpose
Adding a Boundary error message, to prevent publish from crashing on updates with errors.
Related to https://buffer.atlassian.net/browse/PUB-1159

### CODE REVIEW

File: packages/error-boundary/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@bufferapp/publish-error-boundary",
+  "version": "2.0.0",
+  "description": "Allows to display error within Publish",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint . --ignore-pattern coverage node_modules"
+  },
+  "author": "hharnisc@gmail.com",
```
Comment (karelm) [inline packages/error-boundary/package.json]: @MayaUribe would you be able to fix up this package.json please?

File: packages/error-boundary/components/ErrorBoundary/index.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  <Button
+    secondary
+    onClick={() => { this.onRefresh(); }}
+  >
+    <div style={buttonWrapper}>
+      <RefreshIcon color={'curiousBlue'} size={{ width: '14px' }} />
+      <span style={buttonText}>Refresh</span>
+    </div>
+  </Button>
+  */
```
Comment (karelm) [inline packages/error-boundary/components/ErrorBoundary/index.jsx]: would you be able to remove this commented out section please?

File: packages/error-boundary/package.json (hunk 1 above)
Comment (MayaUribe) [reply packages/error-boundary/package.json]: Hey there @karelm I actually ended up deleting this package and creating a fallback component where the ErrorBoundary already takes place.

File: packages/error-boundary/components/ErrorBoundary/index.jsx (hunk 2 above)
Comment (MayaUribe) [reply packages/error-boundary/components/ErrorBoundary/index.jsx]: Hey there @karelm sorry about that, I was doing some tests there, but I already cleaned up the code.

**Comment by MayaUribe** [comment]
> This looks great @MayaUribe! LGTM! 👌
> 
> The only tweak I might suggest would be to import the CSS in the JS like
> 
> ```js-jsx
> import style from './errorBoundary.css`;` 
> ```
> 
> Then you can apply it to the divs/elements like:
> 
> ```js-jsx
> <div style={style.nameOfClass}`> ... </div>
> ```
> 
> Then we could remove the `global()`s in the CSS. The class names would also have to be converted to `camelCase` or you could access them with `style['name-of-class']`. Does that make sense? I say this only since globals can be tricky and cause conflicts sometimes :)

Awesome!! great idea, will update that for sure, makes perfect sense

**Comment by hamstu** [comment]
Thank you @MayaUribe! 🙏

**Pull request merged**